cleanup-passes/travis.ts

```diff
@@ -73,71 +91,68 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
     }
   }
 
-  // use ubuntu trusty
-  travis.dist = 'trusty';
-  travis.sudo = 'required';
+  // Ensure all variants are being tested across all browsers
+  const script = [
+    'xvfb-run polymer test',
+    '>-\nif [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then polymer test -s \'default\';\nfi'
+  ];
+  if (Array.isArray(travis.script)) {
+    const scriptEqual = travis.script.reduce(
+      (acc, s, idx) => { return acc && (script[idx] === s) }, true);
```

Comment:
Same pattern is here as above comment. I would propose to factor out a function that does the deep equal (https://stackoverflow.com/a/16436975) to improve readibility